Makefile

```diff
@@ -9,6 +9,14 @@ install: ## Install the dependencies
 develop: ## Install the test and dev dependencies
 	python3 -m pip install -e .[test,dev]
 
+.PHONY: pretty-templates
+pretty-templates: ## Prettify template files
+	djlint umap/templates --reformat
+
+.PHONY: lint-templates
```

Comment:
AFAIK, you don't need to add .PHONY if the command name does not match with a local path (like `build/`)